### ISSUE TITLE
Action "Building Docker image" fails in forks of this repo

### ISSUE THREAD

**Issue by sebix** [certtools/intelmq#2157]
The docker action fails if the credentials are not available:
```
Run docker/login-action@v1
(node:2702) [DEP0005] DeprecationWarning: Buffer() is deprecated due to security and usability issues. Please use the Buffer.alloc(), Buffer.allocUnsafe(), or Buffer.from() methods instead.
Error: Username and password required
```

Expected behaviour:
- The action does not fail just passes if the creds are not given

**Comment by waldbauer-certat** [comment]
I'd suggest to keep this message, because it seems that there is something weird going on. It only runs in this repository, so we should keep track of pushing updates to dockerhub.

**Comment by sebix** [comment]
> I'd suggest to keep this message, because it seems that there is something weird going on.

Yes, but impossible to fix for all contributors except you. But all contributors get the error in their own forks / PRs.

> It only runs in this repository,

It also runs in all forks.

**Comment by waldbauer-certat** [comment]
> > I'd suggest to keep this message, because it seems that there is something weird going on.
> 
> Yes, but impossible to fix for all contributors except you. But all contributors get the error in their own forks / PRs.

I cant either, only @aaronkaplan is able to setup secrets in this repository.

> 
> > It only runs in this repository,
> 
> It also runs in all forks.

Strange, repository.name is set to `certtools/intelmq`, have to check if github changed such behaviour in the latest changes.

**Comment by sebix** [comment]
> > > I'd suggest to keep this message, because it seems that there is something weird going on.
> > 
> > 
> > Yes, but impossible to fix for all contributors except you. But all contributors get the error in their own forks / PRs.
> 
> I cant either, only @aaronkaplan is able to setup secrets in this repository.

This issue does not appear in certtools/intelmq, but in forks of it.
E.g. https://github.com/sebix/intelmq/runs/5234591929?check_suite_focus=true
```
Run docker/login-action@v1
(node:2702) [DEP0005] DeprecationWarning: Buffer() is deprecated due to security and usability issues. Please use the Buffer.alloc(), Buffer.allocUnsafe(), or Buffer.from() methods instead.
Error: Username and password required
```
> 
> > > It only runs in this repository,
> > 
> > 
> > It also runs in all forks.
> 
> Strange, repository.name is set to `certtools/intelmq`, have to check if github changed such behaviour in the latest changes.

See above.

**Comment by sebix** [comment]
This still fails
https://github.com/certtools/intelmq/actions/runs/7784446907

**Comment by kamil-certat** [comment]
In main repo it requires new credentials, I've sent them last week to @aaronkaplan. When it comes to forks, it should be disabled there. But - the PR integrating subprojects introduces a new Docker, including using Github Repositories, so I won't invest in additional changes here besides new credentials.

**Comment by kamil-certat** [comment]
Apparently new credentials have been already set, but there was a one more problem on my side. It's now fixed and CI works

**Comment by sebix** [comment]
Still fails in forks

```
Login to DockerHub
Run docker/login-action@v2
Error: Username and password required
```

**Comment by aaronkaplan** [comment]
Yeah, I assume the forks don;t copy the credentials (?)

**Comment by sebix** [comment]
And this is very good, otherwise **anyone** could upload docker images in the name of certat
